packages/ocaml-variants/ocaml-variants.4.11.1+BER/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+synopsis: "BER-N111 MetaOCaml"
+maintainer: "yallop@gmail.com"
+depends: [
+  "ocaml" {= "4.11.1" & post}
+  "base-unix" {post}
+  "base-bigarray" {post}
+  "base-threads" {post}
+  "base-metaocaml-ocamlfind" {post}
+]
+conflict-class: "ocaml-core-compiler"
+flags: compiler
+setenv: CAML_LD_LIBRARY_PATH = "%{lib}%/stublibs"
+build: [
+  ["./configure" "-prefix" "%{prefix}%"]
+    {os != "openbsd" & os != "freebsd" & os != "macos"}
+  ["./configure" "-prefix" "%{prefix}%" "-cc" "cc" "-aspp" "cc -c"]
```

Comment:
@yallop: see https://github.com/ocaml/opam-repository/issues/17385
Should we update this file to use 
```
["./configure" "-prefix" "%{prefix}%" "CC=cc" "-aspp" "cc -c"]
```
instead?

Reply:
@meri: that sounds good to me.  See https://github.com/ocaml/opam-repository/pull/17386

Reply:
Thanks!